### DISCUSSION TITLE
Command Palette in 1.1.0.RC10 - wrong position of suggestion list?

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#1093] [Q&A]
Hi,
something seems to have changed with the Command Palette from RC04 to RC10 with the 

The command-palette-suggestions are now no longer placed relative to the command-palette-input. It seems the element is now positioned relative to the man window. If I change the left menue-bar than the command-palette-suggestions position also changed.

See: 

![Peek 2023-08-20 15-04](https://github.com/eclipse-glsp/glsp/assets/1339124/a35a8525-babe-4a1a-b024-ba09c3e08588)

**Comment by rsoika** [comment]
I am not the expert in CSS but I think the problem is the attribute `position: absolute;` assigned to the element `autocomplete command-palette-suggestions`
If I remove this attribute or change it to `fixed` it may propabyl work again correctly.
But I can not really verifiy this as the component is constructed by some javascript dynamically.

**Comment by planger** [comment]
Hi @rsoika, thank you for reporting this! I can reproduce the issue and it indeed is a bug. I've created https://github.com/eclipse-glsp/glsp/issues/1094 to track this.

**Comment by tortmayr** [comment]
@rsoika Thanks for catching this. This bug was introduced when updating to a newer sprotty version. I provided a fix for this directly in sprotty. As you already mentioned this is fixed by adjusting the style position to "fixed".
https://github.com/eclipse-sprotty/sprotty/pull/372
